crates/router/src/routes/payment_methods.rs

```diff
@@ -30,19 +27,13 @@ use crate::{
         storage::payment_method::PaymentTokenData,
     },
 };
-#[cfg(all(
-    any(feature = "v1", feature = "v2", feature = "olap", feature = "oltp"),
-    not(feature = "customer_v2")
-))]
+#[cfg(feature = "v1")]
```

Comment:
Nit:
This should have been:
```rust
#[cfg(any(feature = "v1", feature = "olap", feature = "oltp"))]
```

And we've removed the `customer_v2` feature flag here as well.